.github/workflows/full.yml

```diff
@@ -83,21 +86,19 @@ jobs:
         id: settag
         working-directory: .github/workflows
         run: ./pipeline_set_output_tag.sh $GITHUB_REF_TYPE ${{ github.ref_name }} ${{ github.sha }} $GITHUB_BASE_REF
-
         # if tag isn't the version set in package.json, job will fail
       - name: Check tag version correctness
         if: github.ref_type == 'tag'
         working-directory: .github/workflows
         run: |
           ./check_version.sh ${{ steps.settag.outputs.version }} ../../${{ env.CRUX_WORKING_DIRECTORY }}
           ./check_version.sh ${{ steps.settag.outputs.version }} ../../${{ env.CRUX_UI_WORKING_DIRECTORY }}
-
   # agents scope
   go_lint:
     runs-on: ubuntu-22.04
     needs: gather_changes
     container:
-      image: ghcr.io/dyrector-io/dyrectorio/alpine-proto:3.17-3
+      image: ghcr.io/dyrector-io/dyrectorio/alpine-proto:3.17-4
```

Comment:
Please use a constant instead of these; that way, in the future, we won't need to update all the variables.

Reply:
While defining these can be possible, but rather hacky and work intensive. You cant use expressions there as it turned out.

Reply:
We talked about this IRL, we gonna let it slide for now.